src/ports/postgres/modules/knn/test/knn.sql_in

```diff
@@ -0,0 +1,41 @@
+m4_include(`SQLCommon.m4')
+/* -----------------------------------------------------------------------------
+ * Test knn.
+ *
+ * FIXME: Verify results
```

Comment:
This file is used when you run the install-check. Since the dataset is small you can calculate the correct results by hand (or using some other knn implementation from python, R etc.) and then run an assertion function to ensure the result is correct. 

Since many functions are interconnected, using an install check helps us to identify problems faster. Assume that somebody changed the `squared_dist_norm2` function implementation for  some reason and it started to give incorrect results. This will cause the knn install-check to fail and lead us to more investigation.